src/main/java/com/kryptnostic/conductor/rpc/serializers/AddSchemasToTypeStreamSerializer.java

```diff
@@ -0,0 +1,46 @@
+package com.kryptnostic.conductor.rpc.serializers;
+
+import java.io.IOException;
+import java.util.Set;
+
+import org.apache.olingo.commons.api.edm.FullQualifiedName;
+import org.springframework.stereotype.Component;
+
+import com.dataloom.edm.schemas.processors.AddSchemasToType;
+import com.dataloom.hazelcast.StreamSerializerTypeIds;
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+import com.kryptnostic.rhizome.hazelcast.serializers.SetStreamSerializers;
+import com.kryptnostic.rhizome.pods.hazelcast.SelfRegisteringStreamSerializer;
+
+@Component
+public class AddSchemasToTypeStreamSerializer implements SelfRegisteringStreamSerializer<AddSchemasToType> {
+    @Override
+    public void write( ObjectDataOutput out, AddSchemasToType object ) throws IOException {
+        SetStreamSerializers.serialize( out, object.getSchemas(), ( FullQualifiedName schema ) -> {
+            FullQualifiedNameStreamSerializer.serialize( out, schema );
+        } );
+    }
+
+    @Override
+    public AddSchemasToType read( ObjectDataInput in ) throws IOException {
+        Set<FullQualifiedName> schemas = SetStreamSerializers.deserialize( in, ( ObjectDataInput dataInput ) -> {
```

Comment:
`SetStreamSerializers.deserialize( in, FullQualifiedNameStreamSerializer::deserialize )` ?